### ISSUE TITLE
Illegal Instruction (core dumped) Fedora 35

### ISSUE THREAD

**Issue by lvto2000** [mozilla/TTS#788]
When I run any **tts** command to do anything I get an illegal instruction error.  I have a fresh install using virtualenv with python 3.7.   Here is my **pip show tts**

`(VIRTUALENV-1) [root@fedora TTS]# pip show tts
Name: TTS
Version: 0.0.9.2
Summary: Text to Speech with Deep Learning
Home-page: https://github.com/mozilla/TTS
Author: Eren Gölge
Author-email: egolge@mozilla.com
License: MPL-2.0
Location: /root/VIRTUALENV-1/lib64/python3.7/site-packages
Requires: bokeh, cardboardlint, cython, flask, gdown, inflect, librosa, matplotlib, nose, numba, numpy, phonemizer, Pillow, pylint, pysbd, pyworld, pyyaml, scipy, soundfile, tensorboardX, tensorflow, torch, tqdm, umap-learn, unidecode
Required-by:
(VIRTUALENV-1) [root@fedora TTS]# tts
Illegal instruction (core dumped)
(VIRTUALENV-1) [root@fedora TTS]#
`

Here is my journalctl output right when the core dump happens. 

`Sep 09 20:51:46 fedora audit[1952]: ANOM_ABEND auid=0 uid=0 gid=0 ses=1 subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 pid            =1952 comm="tts" exe="/usr/bin/python3.7" sig=4 res=1
Sep 09 20:51:46 fedora kernel: traps: tts[1952] trap invalid opcode ip:7f090c77e110 sp:7ffc75415ed8 error:0 in _pywrap_tensorflow_inte            rnal.so[7f08fec00000+1d6ed000]
Sep 09 20:51:46 fedora systemd[1]: Started Process Core Dump (PID 1977/UID 0).
Sep 09 20:51:46 fedora audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=s            ystemd-coredump@12-1977-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Sep 09 20:51:48 fedora abrt-dump-journal-oops[742]: abrt-dump-journal-oops: Found oopses: 1
Sep 09 20:51:48 fedora abrt-dump-journal-oops[742]: abrt-dump-journal-oops: Creating problem directories
Sep 09 20:51:48 fedora abrt-server[1979]: Can't find a meaningful backtrace for hashing in '.'
Sep 09 20:51:48 fedora abrt-server[1979]: Preserving oops '.' because DropNotReportableOopses is 'no'
Sep 09 20:51:48 fedora systemd[1]: Started dbus-:1.2-org.freedesktop.problems@9.service.
Sep 09 20:51:48 fedora audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=d            bus-:1.2-org.freedesktop.problems@9 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Sep 09 20:51:48 fedora abrt-notification[2000]: [🡕] System encountered a non-fatal error in ??()
Sep 09 20:51:49 fedora abrt-dump-journal-oops[742]: Reported 1 kernel oopses to Abrt
Sep 09 20:51:51 fedora systemd-coredump[1978]: [🡕] Process 1952 (tts) of user 0 dumped core.

                                                    Stack trace of thread 1952:
                                               #0  0x00007f090c77e110 n/a (/root/VIRTUALENV-1/lib64/python3.7/site-packages/tensorflow            /python/_pywrap_tensorflow_internal.so + 0xdb7e110)
                                               #1  0x00007f08fd2d532e n/a (/root/VIRTUALENV-1/lib64/python3.7/site-packages/tensorflow            /libtensorflow_framework.so.2 + 0x6d532e)
                                               #2  0x00007f09fdf7613e call_init (ld-linux-x86-64.so.2 + 0xa13e)
                                               #3  0x00007f09fdf7622c _dl_init (ld-linux-x86-64.so.2 + 0xa22c)
                                               #4  0x00007f09fd995db5 _dl_catch_exception (libc.so.6 + 0x195db5)
                                               #5  0x00007f09fdf7cc4e dl_open_worker (ld-linux-x86-64.so.2 + 0x10c4e)
                                               #6  0x00007f09fd995d58 _dl_catch_exception (libc.so.6 + 0x195d58)
                                               #7  0x00007f09fdf7cfdf _dl_open (ld-linux-x86-64.so.2 + 0x10fdf)
                                               #8  0x00007f09fd89b88c dlopen_doit (libc.so.6 + 0x9b88c)
                                               #9  0x00007f09fd995d58 _dl_catch_exception (libc.so.6 + 0x195d58)
                                               #10 0x00007f09fd995e23 _dl_catch_error (libc.so.6 + 0x195e23)
                                               #11 0x00007f09fd89b35e _dlerror_run (libc.so.6 + 0x9b35e)
                                               #12 0x00007f09fd89b941 dlopen@GLIBC_2.2.5 (libc.so.6 + 0x9b941)
                                               #13 0x00007f09fdd85d46 _PyImport_FindSharedFuncptr (libpython3.7m.so.1.0 + 0x185d46)
                                               #14 0x00007f09fdd8503b _PyImport_LoadDynamicModuleWithSpec (libpython3.7m.so.1.0 + 0x18            503b)
                                               #15 0x00007f09fdd84e23 _imp_create_dynamic (libpython3.7m.so.1.0 + 0x184e23)
                                               #16 0x00007f09fdd02af9 _PyMethodDef_RawFastCallDict (libpython3.7m.so.1.0 + 0x102af9)
                                               #17 0x00007f09fdd02a37 _PyCFunction_FastCallDict (libpython3.7m.so.1.0 + 0x102a37)
                                               #18 0x00007f09fdcffc43 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xffc43)
                                               #19 0x00007f09fdcf9179 _PyEval_EvalCodeWithName (libpython3.7m.so.1.0 + 0xf9179)
                                               #20 0x00007f09fdd095ea _PyFunction_FastCallKeywords (libpython3.7m.so.1.0 + 0x1095ea)
                                               #21 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #22 0x00007f09fdcfece2 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfece2)
                                               #23 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #24 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #25 0x00007f09fdcfa9e8 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa9e8)
                                               #26 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #27 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #28 0x00007f09fdcfa884 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa884)
                                               #29 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #30 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #31 0x00007f09fdcfa884 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa884)
                                               #32 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #33 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #34 0x00007f09fdcfa884 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa884)
                                               #35 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #36 0x00007f09fdd08f58 object_vacall (libpython3.7m.so.1.0 + 0x108f58)
                                               #37 0x00007f09fdd159bd _PyObject_CallMethodIdObjArgs (libpython3.7m.so.1.0 + 0x1159bd)
                                               #38 0x00007f09fdd13538 PyImport_ImportModuleLevelObject (libpython3.7m.so.1.0 + 0x11353            8)
                                               #39 0x00007f09fdcfce73 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfce73)
                                               #40 0x00007f09fdcf9179 _PyEval_EvalCodeWithName (libpython3.7m.so.1.0 + 0xf9179)
                                               #41 0x00007f09fdcf8e6d PyEval_EvalCodeEx (libpython3.7m.so.1.0 + 0xf8e6d)
                                               #42 0x00007f09fdd6882f PyEval_EvalCode (libpython3.7m.so.1.0 + 0x16882f)
                                               #43 0x00007f09fdd6c29c builtin_exec (libpython3.7m.so.1.0 + 0x16c29c)
                                               #44 0x00007f09fdd02af9 _PyMethodDef_RawFastCallDict (libpython3.7m.so.1.0 + 0x102af9)
                                               #45 0x00007f09fdd02a37 _PyCFunction_FastCallDict (libpython3.7m.so.1.0 + 0x102a37)
                                               #46 0x00007f09fdcffc43 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xffc43)
                                               #47 0x00007f09fdcf9179 _PyEval_EvalCodeWithName (libpython3.7m.so.1.0 + 0xf9179)
                                               #48 0x00007f09fdd095ea _PyFunction_FastCallKeywords (libpython3.7m.so.1.0 + 0x1095ea)
                                               #49 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #50 0x00007f09fdcfece2 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfece2)
                                               #51 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #52 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #53 0x00007f09fdcfa9e8 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa9e8)
                                               #54 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #55 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #56 0x00007f09fdcfa884 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa884)
                                               #57 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #58 0x00007f09fdd018dc call_function (libpython3.7m.so.1.0 + 0x1018dc)
                                               #59 0x00007f09fdcfa884 _PyEval_EvalFrameDefault (libpython3.7m.so.1.0 + 0xfa884)
                                               #60 0x00007f09fdd0991a function_code_fastcall (libpython3.7m.so.1.0 + 0x10991a)
                                               #61 0x00007f09fdd08f58 object_vacall (libpython3.7m.so.1.0 + 0x108f58)
                                               #62 0x00007f09fdd159bd _PyObject_CallMethodIdObjArgs (libpython3.7m.so.1.0 + 0x1159bd)
                                               #63 0x00007f09fdd13538 PyImport_ImportModuleLevelObject (libpython3.7m.so.1.0 + 0x11353            8)
Sep 09 20:51:51 fedora systemd[1]: systemd-coredump@12-1977-0.service: Deactivated successfully.
Sep 09 20:51:51 fedora audit[1]: SERVICE_STOP pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='unit=sy            stemd-coredump@12-1977-0 comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
Sep 09 20:51:51 fedora systemd[1]: systemd-coredump@12-1977-0.service: Consumed 3.903s CPU time.
Sep 09 20:51:51 fedora audit: BPF prog-id=0 op=UNLOAD
Sep 09 20:51:51 fedora audit: BPF prog-id=0 op=UNLOAD
Sep 09 20:51:51 fedora audit: BPF prog-id=0 op=UNLOAD
Sep 09 20:51:52 fedora abrt-server[2008]: Interpreter crashed, but no packaged script detected: '/root/VIRTUALENV-1/bin/python /root/V            IRTUALENV-1/bin/tts'
Sep 09 20:51:52 fedora abrt-server[2008]: 'post-create' on '/var/spool/abrt/ccpp-2024-09-09-20:51:51.869450-1952' exited with 1
Sep 09 20:51:52 fedora abrt-server[2008]: Deleting problem directory '/var/spool/abrt/ccpp-2024-09-09-20:51:51.869450-1952'`

Here is my PIP list.
`(VIRTUALENV-1) [root@fedora TTS]# pip list
Package                  Version
------------------------ -----------
absl-py                  2.1.0
astroid                  2.5
astunparse               1.6.3
attrs                    24.2.0
audioread                3.0.1
Babel                    2.14.0
beautifulsoup4           4.12.3
bokeh                    1.4.0
cachetools               5.5.0
cardboardlint            1.3.0
certifi                  2024.8.30
cffi                     1.15.1
charset-normalizer       3.3.2
click                    8.1.7
clldutils                3.19.0
colorama                 0.4.6
colorlog                 6.8.2
csvw                     3.1.3
cycler                   0.11.0
Cython                   3.0.11
decorator                5.1.1
dlinfo                   1.2.1
filelock                 3.12.2
Flask                    2.2.5
fonttools                4.38.0
gast                     0.3.3
gdown                    4.7.3
google-auth              2.34.0
google-auth-oauthlib     0.4.6
google-pasta             0.2.0
grpcio                   1.62.3
h5py                     2.10.0
idna                     3.8
importlib-metadata       4.13.0
importlib-resources      5.12.0
inflect                  6.0.5
isodate                  0.6.1
isort                    4.3.21
itsdangerous             2.1.2
Jinja2                   3.1.4
joblib                   1.3.2
jsonschema               4.17.3
Keras-Preprocessing      1.1.2
kiwisolver               1.4.5
language-tags            1.2.0
lazy-object-proxy        1.9.0
librosa                  0.7.2
llvmlite                 0.31.0
lxml                     5.3.0
Markdown                 3.4.4
MarkupSafe               2.1.5
matplotlib               3.5.3
mccabe                   0.6.1
nose                     1.3.7
numba                    0.48.0
numpy                    1.17.5
nvidia-cublas-cu11       11.10.3.66
nvidia-cuda-nvrtc-cu11   11.7.99
nvidia-cuda-runtime-cu11 11.7.99
nvidia-cudnn-cu11        8.5.0.96
oauthlib                 3.2.2
opt-einsum               3.3.0
packaging                24.0
phonemizer               3.2.1
Pillow                   9.5.0
pip                      24.0
pkgutil_resolve_name     1.3.10
protobuf                 3.20.3
pyasn1                   0.5.1
pyasn1-modules           0.3.0
pycparser                2.21
pydantic                 1.10.18
pylatexenc               2.10
pylint                   2.5.3
pyparsing                3.1.4
pyrsistent               0.19.3
pysbd                    0.3.4
PySocks                  1.7.1
python-dateutil          2.9.0.post0
pytz                     2024.1
pyworld                  0.3.4
PyYAML                   6.0.1
rdflib                   6.3.2
regex                    2024.4.16
requests                 2.31.0
requests-oauthlib        2.0.0
resampy                  0.3.1
rfc3986                  1.5.0
rsa                      4.9
scikit-learn             1.0.2
scipy                    1.7.3
segments                 2.2.1
setuptools               57.4.0
six                      1.16.0
soundfile                0.12.1
soupsieve                2.4.1
tabulate                 0.9.0
tensorboard              2.11.2
tensorboard-data-server  0.6.1
tensorboard-plugin-wit   1.8.1
tensorboardX             2.6.2.2
tensorflow               2.3.1
tensorflow-estimator     2.3.0
termcolor                2.3.0
threadpoolctl            3.1.0
toml                     0.10.2
torch                    1.13.1
tornado                  6.2
tqdm                     4.66.5
TTS                      0.0.9.2
typed-ast                1.4.3
typing_extensions        4.7.1
umap-learn               0.4.6
Unidecode                0.4.20
uritemplate              4.1.1
urllib3                  2.0.7
Werkzeug                 2.2.3
wheel                    0.36.2
wrapt                    1.12.1
zipp                     3.15.0
(VIRTUALENV-1) [root@fedora TTS]#
`
